meta.go

```diff
@@ -54,6 +56,82 @@ func getMeta(key string, metaSpace string, output io.Writer) error {
 	return nil
 }
 
+func fetchMetaValue(key string, meta interface{}) (string, interface{}) {
+	var result interface{}
+	for position, char := range key {
+		if string([]rune{char}) == "[" {
+			// Value is array with index
+			var i int
+			for i = position + 1; ; i++ {
+				_, err := strconv.Atoi(string(key[i])) // Check the next char is integer
+				if err != nil {
+					break
+				}
+			}
+			metaIndex, _ := strconv.Atoi(key[position+1 : i]) // e.g. if array[10], get "10"
+			shortenKey := key[i+1:]                           // Remove bracket[num] from key
+			metaValue := reflect.ValueOf(meta)
+			// convert type interface -> Value -> map[string]interface{}
+			metaMap := make(map[string]interface{})
+			if metaValue.Kind() == reflect.Map {
+				for _, k := range metaValue.MapKeys() {
+					pk, _ := k.Interface().(string)
```

Comment:
Maybe it means previous Meta key but `k` is also previous Meta key... It is only type conversion.